notebooks/ConsPrefShockModel.py

```diff
@@ -0,0 +1,267 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.2'
+#       jupytext_version: 1.2.4
+#   kernelspec:
+#     display_name: Python 2
+#     language: python
+#     name: python2
```

Comment:
python2 in kernelspec is messing with Travis (cuz travis is only running on python3), make sure to change your kernel to python3 manually locally and then push again, jupyter lab/notebook will automatically take care of the metadata.

Reply:
I've pushed a change with a different kernel name. I have a local kernel named econ-ark which has econ-ark installed, separate from my local, more general Python 3 kernel. This econ-ark kernel uses python 3.

In general, I think it's possible for developers to have many different jupyter kernels, and for these kernels to have names that are not merely designations of the version of python used by them (Python 3 is ambiguous anyway).

I'm not resisting the idea of mandating that all DemARK developers have an identical Jupyter kernel with an identifical name if that's what's necessary to make the contiguous integration/testing to work right.

But I'm (a) doubtful if this is truly the best strategy (is there any way for Travis to ignore the kernelspec and use the same kernel all the time? this wasn't a problem before...) and (b) if it _is_ the best strategy, I think it needs to be specifically documented. There's a lot of degrees of freedom here to mess up on.

In the short term, I'll try saving and running the notebook in my local Python 3 kernel next. I expect this to fail, because I don't believe that kernel has econ-ark installed. (It's more bare bones).